src/nvim/globals.h

```diff
@@ -332,8 +332,8 @@ EXTERN int garbage_collect_at_exit INIT(= false);
 #define SID_LUA         -7      // for Lua scripts/chunks
 #define SID_API_CLIENT  -8      // for API clients
 
-// ID of script being sourced or was sourced to define the current function.
-EXTERN scid_T current_SID INIT(= 0);
+// Script CTX being sourced or was sourced to define the current function.
+EXTERN sctx_T current_sctx INIT(= { 0 COMMA 0 });
```

Comment:
I wonder if we actually need `COMMA`. It's from https://github.com/vim/vim/commit/ac77aec4daea8d73468fcf4690cb4ccab1d807ed . 

We use `,` (literal comma) when defining `g_stats`, and we don't get a clang warning.